src/components/Dialog/Dialog.js

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from "react";
+import { Portal } from "react-portal";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+
+import "./Dialog.scss"
+
+const Dialog = ({title, body, handleClose}) => {
```

Comment:
I suggest you to review a topic about using `children` in React - https://codeburst.io/a-complete-guide-to-props-children-in-react-c315fab74e7c.

Reply:
Fixed in my next commit